src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsallocations/domain/UnallocatedCaseDetails.kt

```diff
@@ -77,7 +83,11 @@ data class UnallocatedCaseDetails @JsonCreator constructor(
         UnallocatedCaseDocument.from(deliusCaseView.preConvictionDocument),
         deliusCaseView.mainAddress,
         deliusCaseView.sentence.length,
-        case.convictionNumber
+        case.convictionNumber,
+        unallocatedCaseRisks?.roshRisk?.overallRisk,
+        unallocatedCaseRisks?.rsr?.level,
+        unallocatedCaseRisks?.ogrs?.score,
+        unallocatedCaseRisks?.activeRegistrations?.joinToString(", ") { it.type }.takeUnless { it.isNullOrEmpty() }
```

Comment:
what does this takeUnless do here? this happens before the joinToString?

Reply:
to prevent empty string

Reply:
This will be changed to `unallocatedCaseRisks?.activeRegistrations?.takeUnless { it.isEmpty() }?.joinToString(", ") { it.type }`